content/en/_redirects

```diff
@@ -96,6 +96,9 @@ docs/started/requirements/                    /docs/started/getting-started/
 # Remove Kubeflow installation on existing EKS cluster
 /docs/aws/deploy/existing-cluster/             /docs/distributions/aws/deploy/install-kubeflow/
 
+# Remove examples docs
+/docs/examples/*                                /docs
```

Comment:
As we are moving things from `/docs/examples` to `/docs/started/kubeflow-examples` could you add the latter in this line? 

Thank you for this!!

Reply:
Ok